### ISSUE TITLE
When using latest commit of typescript specified in config, built in types are not found

### ISSUE THREAD

**Issue by 2426021684** [TypeStrong/atom-typescript#907]
If I chose I custom typescriptServices.js to be the latest from Typescript head built by running jake I am unable to use built in types like `HTMLElement`. I get errors like `Cannot find name 'HTMLElement'`.

ls of /lib in working version (old):
```
README.md
lib.core.d.ts
lib.core.es6.d.ts
lib.core.es7.d.ts
lib.d.ts
lib.dom.d.ts
lib.es6.d.ts
lib.es7.d.ts
lib.scriptHost.d.ts
lib.webworker.d.ts
tsc.js
tsserver.js
typescript.d.ts
typescript.js
typescriptServices.d.ts
typescriptServices.js
```
ls of /lib in broken version (new):
```
README.md
lib.core.d.ts
lib.core.es6.d.ts
lib.core.es7.d.ts
lib.d.ts
lib.dom.d.ts
lib.dom.iterable.d.ts
lib.es2015.array.d.ts
lib.es2015.collection.d.ts
lib.es2015.d.ts
lib.es2015.function.d.ts
lib.es2015.generator.d.ts
lib.es2015.iterable.d.ts
lib.es2015.math.d.ts
lib.es2015.number.d.ts
lib.es2015.object.d.ts
lib.es2015.promise.d.ts
lib.es2015.proxy.d.ts
lib.es2015.reflect.d.ts
lib.es2015.regexp.d.ts
lib.es2015.string.d.ts
lib.es2015.symbol.d.ts
lib.es2015.symbol.wellknown.d.ts
lib.es2016.array.include.d.ts
lib.es2016.d.ts
lib.es5.d.ts
lib.es6.d.ts
lib.es7.d.ts
lib.full.es2015.d.ts
lib.scriptHost.d.ts
lib.webworker.d.ts
tsc.js
tsserver.js
typescript.d.ts
typescript.js
typescriptServices.d.ts
typescriptServices.js
```

I believe this change in the structure of type definitions is causing this issue.

**Comment by 2426021684** [comment]
Same with lastest from head (not my fork). 
`Cannot find name 'HTMLElement'`
There are similiar error messages for other built in types.

**Comment by 2426021684** [comment]
This only happens if I build typescript myself and use the typescriptServices in /built. If I use the one in /lib (prebuilt) it works fine. I believe this due to a change in the typescript compiler in a very recent commit.

**Comment by 2426021684** [comment]
ls of working version (old):
```
README.md
lib.core.d.ts
lib.core.es6.d.ts
lib.core.es7.d.ts
lib.d.ts
lib.dom.d.ts
lib.es6.d.ts
lib.es7.d.ts
lib.scriptHost.d.ts
lib.webworker.d.ts
tsc.js
tsserver.js
typescript.d.ts
typescript.js
typescriptServices.d.ts
typescriptServices.js
```
ls of broken version (non working):
```
README.md
lib.core.d.ts
lib.core.es6.d.ts
lib.core.es7.d.ts
lib.d.ts
lib.dom.d.ts
lib.dom.iterable.d.ts
lib.es2015.array.d.ts
lib.es2015.collection.d.ts
lib.es2015.d.ts
lib.es2015.function.d.ts
lib.es2015.generator.d.ts
lib.es2015.iterable.d.ts
lib.es2015.math.d.ts
lib.es2015.number.d.ts
lib.es2015.object.d.ts
lib.es2015.promise.d.ts
lib.es2015.proxy.d.ts
lib.es2015.reflect.d.ts
lib.es2015.regexp.d.ts
lib.es2015.string.d.ts
lib.es2015.symbol.d.ts
lib.es2015.symbol.wellknown.d.ts
lib.es2016.array.include.d.ts
lib.es2016.d.ts
lib.es5.d.ts
lib.es6.d.ts
lib.es7.d.ts
lib.full.es2015.d.ts
lib.scriptHost.d.ts
lib.webworker.d.ts
tsc.js
tsserver.js
typescript.d.ts
typescript.js
typescriptServices.d.ts
typescriptServices.js
```

**Comment by 2426021684** [comment]
Does anyone know of any way to fix this?

**Comment by 2426021684** [comment]
@basarat

**Comment by 2426021684** [comment]
This has been fixed in 8.8.0
